src/ActionHistory.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import Action from './Action';
+
+const ActionHistory = (props) => {
+  return (
+    <div className="side-bar action-history">
+      <h3>Action History</h3>
+      {props.actions.length > 0 ? null : 'Perform an action, jagweed.'}
```

Comment:
When checking if an array has items, you can use the shorthand and just write `props.actions.length ? null : '...'` Since length returns a truthy value when there is items in the array.